7_examples/20_annotations.tex

```diff
@@ -18,69 +18,69 @@ \section{Приклад застосування для не супермоду
 
 Запустимо процедуру субградієнтного спуску. Виберемо початкову репараметризацію
-\begin{equation}
+\begin{equation*}
     \varphi_{tt'}(k)=0,\forall k\in K, t\in T, t'\in N_t,
-\end{equation}
+\end{equation*}
 і крок субградієнтного спуску 
-\begin{equation}
+\begin{equation*}
     \gamma_i=\left(\frac{x}{2}+1\right)/i.
-\end{equation}
+\end{equation*}
 Виконаємо 2 ітерації субградієнтного спуску й отримаємо значення репараметризації 
-\begin{equation}
+\begin{equation*}
     \begin{aligned}
        \varphi_{t_3t_4}(0)=\left(\frac{x}{2}+1\right)/2, \varphi_{t_3t_4}(2)=-\left(\frac{x}{2}+1\right)/2,\\
        \varphi_{t_2t_4}(0)=\left(\frac{x}{2}+1\right)/2, \varphi_{t_2t_4}(2)=-\left(\frac{x}{2}+1\right)/2,\\
        \varphi_{t_4t_3}(0)=-\left(\frac{x}{2}+1\right)/2, \varphi_{t_4t_3}(2)=\left(\frac{x}{2}+1\right)/2,\\
        \varphi_{t_4t_2}(0)=-\left(\frac{x}{2}+1\right)/2, \varphi_{t_4t_2}(2)=\left(\frac{x}{2}+1\right)/2.\\
     \end{aligned}
-\end{equation}
+\end{equation*}
 Всі інші значення репараметризації $\varphi$ не змінилися. За такої репараметризації і 
-\begin{equation}
+\begin{equation*}
     \varepsilon\leq \frac{1}{|T|+2\cdot|\Gamma|+1}\approx 0.077,
-\end{equation}
+\end{equation*}
 задача є $\varepsilon$-узгодженою.
 Значення цільової функції 
-\begin{equation}
+\begin{equation*}
     E(q,g,\varphi)=-x+1.
-\end{equation}
+\end{equation*}
 У кожному об'єкті всі мітки мають скінченну вагу, тому другий крок алгоритму пропустимо.
 Обхід об'єктів графу здійснюємо у порядку $(t_1,t_2,t_3,t_4)$, а обхід міток в 
 об'єкті виконуємо у порядку $(0,1,2)$. Нехай $K_1=\{0\}$, $K_2=\{1,2\}$. Зафіксуємо мітку $0$ в 
 об'єкті $t_1$ та заборонимо інші мітки у цьому об'єкті
-\begin{equation}
+\begin{equation*}
     \begin{aligned}
     q'_{t_1}(0)=0,q'_{t_1}(1)=\infty,q'_{t_1}(2)=\infty,\\
     q'_t(k)=q_t(k),\forall t\in \{t_2,t_3,t_4\}, \forall k\in K.
     \end{aligned}
-\end{equation}
+\end{equation*}
 Використаємо значення $\varphi$, отримані на попередньому кроці. Одразу маємо 
 $\varepsilon$-узгоджений набір вершин та ребер, а також 
-\begin{equation}
+\begin{equation*}
     E(q',g,\varphi)=E(q,g,\varphi)=-x+1.
-\end{equation}
+\end{equation*}
 В об'єкті $t_1$ залишимо мітку $0$, тобто $k^*_{t_1}=0$. Проведемо аналогічну процедуру 
 для всіх інших об'єктів. У результаті отримаємо оптимальну розмітку $k^*$.
-\begin{equation}
+\begin{equation*}
     k^*(t_1)=k^*(t_2)=k^*(t_3)=k^*(t_4)=0,
-\end{equation}
+\end{equation*}
 з
-\begin{equation}
+\begin{equation*}
     G(k^*)=-x+1>E(q,g,\varphi)-1=-x.
-\end{equation}
+\end{equation*}
 
 \section{Результати експериментів}
 
 Як приклад застосування алгоритму було використано задачу знешумлення бінарного зображення 
 \cite{Boykov,Boykov_2,comp_vision,Greig_port}. Зображення з шумом отримується шляхом додавання сильного
-гаусового шуму до початкового зображення (без шуму). 
+гаусового шуму до початкового зображення (без шуму). Приклад зображення на \ref{fig:a2_original}.
```

Comment:
```suggestion
ґауссового шуму до початкового зображення (без шуму). Приклад зображення на \ref{fig:a2_original}.
```

Reply:
https://goroh.pp.ua/%D1%81%D0%BB%D0%BE%D0%B2%D0%BE%D0%B7%D0%BC%D1%96%D0%BD%D0%B0/%D0%B3%D0%B0%D1%83%D1%81%D0%BE%D0%B2%D0%BE%D1%97